src/Moq.Analyzers/AnalyzerReleases.Unshipped.md

```
@@ -1,2 +1,8 @@
 ﻿; Unshipped analyzer release
 ; https://github.com/dotnet/roslyn-analyzers/blob/main/src/Microsoft.CodeAnalysis.Analyzers/ReleaseTrackingAnalyzers.Help.md
+
+### New Rules
+
+Rule ID | Category | Severity | Notes
+--------|----------|----------|-------
+Moq1400 | Moq      | Warning  | SetExplicitMockBehaviorAnalyzer, [Documentation](https://github.com/rjmurillo/moq.analyzers/blob/main/docs/rules/Moq1400.md)
```

Markdownlint

Files should end with a single newline character

(MD047, single-trailing-newline)